tests/expressions/test_queryset_values.py

```diff
@@ -34,6 +35,12 @@ def test_values_expression(self):
             [{"salary": 10}, {"salary": 20}, {"salary": 30}],
         )
 
+    def test_values_expression_containing_percent_sign_deprecation_warns_once(self):
+        msg = "Using percent signs in a column alias is deprecated."
+        with self.assertWarnsMessage(RemovedInDjango70Warning, msg) as cm:
+            Company.objects.values(**{"alias%": F("id")})
+        self.assertEqual(len(cm.warnings), 1)
```

Comment:
Nice idea. Your version was a little more precise, but I think the simple version will do, what say you?